Sources/PrimerSDK/Classes/Data Models/CardNetwork.swift

```diff
@@ -232,7 +232,36 @@ public enum CardNetwork: String, CaseIterable {
     }
     
     public var icon: UIImage? {
-        return nil
+        switch self {
+        case .amex:
+            return UIImage(named: "amex", in: Bundle.primerResources, compatibleWith: nil)
+        case .bancontact:
+            return UIImage(named: "bancontact-icon", in: Bundle.primerResources, compatibleWith: nil)
+        case .diners:
+            return UIImage(named: "genericCard", in: Bundle.primerResources, compatibleWith: nil)
```

Comment:
maybe we should raise a request to have icons for these other ones too. Wdyt?

Reply:
yeah, I looked for them in Figma and they aren't there.